R/deployApp.R

```diff
@@ -75,8 +75,12 @@
 #'   You can find the `appId` in the following places:
 #'   * On shinyapps.io, it's the `id` listed on the applications page.
 #'   * For Posit Connect, it's `guid` from the info tab on the content page.
-#' @param contentCategory Optional; the kind of content being deployed (e.g.
-#'   `"plot"` or `"site"`).
+#' @param appMode Optional; the type of content being deployed (e.g. `"shiny"`).
```

Comment:
Do we need to document all of the different valid `appMode` values?

Reply:
At this point, I was thinking of this as "expert use only". The accepted values are available in the Connect API reference. rsconnect currently generates a subset of these values. Maybe I'll add something like:

> Accepted values include "shiny", "api", "rmd-static", "quarto-static", and "static".  See the `<Posit Connect API Reference>` for a full set of values. Not all servers support all types of content.

Reply:
That looks perfect to me.